pkg/frontend/protocol.go

```diff
@@ -349,6 +359,7 @@ func (mp *MysqlProtocolImpl) SendResponse(resp *Response) error {
 		if err == nil {
 			return mp.sendOKPacket(0, 0, uint16(resp.status), 0, "")
 		}
+		attachAbort := mp.getAbortTransactionErrorInfo()
 		switch myerr := err.(type) {
 		case *moerr.Error:
```

Comment:
Why should we ever get anything other than moerr?

Reply:
As explaining in above. we need attach the new info to the existed error generated by the mo.